products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
I am adding a step for checking and rollback registration in the case of online migration failure lead to broken on redo online migration. but the problem is for sle12sp0, the version SUSEConnect doens't support the rollback registration feature. Is it possible to generate sle12sp0 images with updated SUSEConnect?
Also I wonder konw all the registered images with update or not?
And how many scenarios for registered images? that means how many combinations of system and addons?